src/views/month/single-day.php

```diff
@@ -40,7 +40,7 @@
 	<div class="tribe-events-viewmore">
 		<?php
 
-			$view_all_label = sprintf( _n( 'View 1 %1$s', 'View All %2$s %3$s', $day['total_events'], 'tribe-events-calendar' ), $events_label_singular, $day['total_events'], $events_label_plural );
+			$view_all_label = sprintf( _n( 'View 1 %s', 'View All %s %s', $day['total_events'], 'tribe-events-calendar' ), $events_label_singular, $day['total_events'], $events_label_plural );
```

Comment:
Switching to that format results in the following `$view_all_label` when there are more than 1 events in viewmore:

```
View All Event 4
```